ndcube/ndcube.py

```diff
@@ -709,6 +709,220 @@ def reproject_to(self, target_wcs, algorithm='interpolation', shape_out=None, re
 
         return resampled_cube
 
+    def rebin(self, bin_shape, **kwargs):
```

Comment:
I can't see a note or anything saying that kwargs is passed to another function, that might be helpful?

Reply:
It is said [here](https://github.com/sunpy/ndcube/pull/450/files#diff-929794bb532be1486907b360285433b432f8505ab7e7ff63eca3eab657f7d00dR805).  But this should probably be said up front.

Reply:
Actually it's also said [here](https://github.com/sunpy/ndcube/pull/450/files#diff-929794bb532be1486907b360285433b432f8505ab7e7ff63eca3eab657f7d00dR743).

Reply:
I think it should be added to the parameters section.